src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -85,6 +85,13 @@ abstract class Model implements Arrayable, ArrayAccess, CanBeEscapedWhenCastToSt
      */
     protected $withCount = [];
 
+    /**
+     * The attributes to reload on save.
+     *
+     * @var array<int, string>
+     */
+    protected $attributesToReloadOnSave = [];
```

Comment:
Not sure if it's too generic but maybe just `$reload`?

Reply:
Yeah, I wasn't sure about naming too, `$reload` feels like reloading relations? specially when we have `load` method on eloquent.

Reply:
That's true.